buy_together_app/tests.py

```diff
@@ -1,3 +1,63 @@
-# from django.test import TestCase
+import pytest
+from buy_together_app.models import supplierProduct , Product, Supplier
 
-# Create your tests here.
+supID0 = 123
+supID1 = 222
+supID2 = 127    
+class TestSupProduct:
+
+    @pytest.fixture
+    def creat_product(self):
+        prod0 = Product(qr_code = "AASSD22", product_name = "APPLE", description= "sweet!!")        
+        prod1 = Product(qr_code = "QS#$HHS", product_name = "CARROT", description= "yummy!!")
+        prod2 = Product(qr_code = "AAsdgv66", product_name = "BANANA", description= "great!!")
+        prod0.save()
+        prod1.save()
+        prod2.save()
+        return [prod0, prod1, prod2]
+
+    @pytest.fixture
+    def creat_supplier(self):
+        sup0 = Supplier (user_name = "ali22", first_name = "ali" , last_name = "ab", password = "2345678", access = True)
+        sup1 = Supplier(user_name = "max00", first_name = "Max" , last_name = "hani", password = "123456789", access = True)
+        sup2 = Supplier (user_name = "rawanab", first_name = "rawan" , last_name = "abu", password = "0000000", access = True)
+        sup0.save()
+        sup1.save()
+        sup2.save()     
+        return [sup0, sup1, sup2]
+
+    @pytest.fixture
+    def creat_supProd(self, creat_product , creat_supplier):
+        supprod0 = supplierProduct(suppler_product_id = supID0, user_name = creat_supplier[0], qr_code = creat_product[0], price = 22, quantity = 4) 
```

Comment:
why not use the default id from django in supplier_product_id (right there is id in the erd but you dont need to override it with new one i guess)